app/main/views/frameworks.py

```diff
@@ -93,22 +98,31 @@ def framework_services():
         complete_drafts=list(reversed(complete_drafts)),
         drafts=list(reversed(drafts)),
         declaration_status=declaration_status,
-        g7_status=g7_status,
+        framework=framework,
         **template_data
     ), 200
 
 
-@main.route('/frameworks/g-cloud-7/declaration/<string:section_id>',
-            methods=['GET', 'POST'])
+@main.route('/frameworks/<framework_slug>/declaration', methods=['GET'])
+@main.route('/frameworks/<framework_slug>/declaration/<string:section_id>', methods=['GET', 'POST'])
```

Comment:
The page title at this route (i.e. `/suppliers/frameworks/digital-outcomes-and-specialists/declaration`) is still `G-Cloud 7 supplier declaration – Digital Marketplace` - would be nice to pull out `framework.name` for the titles.